admin_manual/installation/nginx_configuration.rst

```diff
@@ -80,8 +80,10 @@ Nginx Configuration
       rewrite ^/.well-known/host-meta /public.php?service=host-meta last;
       rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json last;
 
-      rewrite ^/.well-known/carddav /remote.php/carddav/ redirect;
-      rewrite ^/.well-known/caldav /remote.php/caldav/ redirect;
+      rewrite ^/.well-known/carddav /remote.php/carddav/ permanent;
+      rewrite ^/.well-known/caldav /remote.php/caldav/ permanent;
+
+      rewrite ^/remote/(.*) /remote.php last;
```

Comment:
This is also something i really don't know if it its needed. Just tested and on Apache as well as on nginx a 404 is caused by this:

```
 curl -i http://127.0.0.1/remote/abc
HTTP/1.1 404 Not Found
Date: Tue, 13 Oct 2015 12:43:31 GMT
Server: Apache
Content-Length: 267
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /remote/abc was not found on this server.</p>
<hr>
<address>Apache Server at 127.0.0.1 Port 80</address>
</body></html>
```